controllers/constant/odlm.go

```diff
@@ -530,7 +530,112 @@ spec:
                 keytool -importcert -file ${CA_DIR}/${cert} -keystore ${TRUSTSTORE_DIR}/keycloak-truststore.jks -storepass changeit -alias ${cert} -noprompt
               done
               echo "Truststore file built, starting Keycloak ..."
-              "/opt/keycloak/bin/kc.sh" "$@" --spi-truststore-file-file=${TRUSTSTORE_DIR}/keycloak-truststore.jks --spi-truststore-file-password=changeit --spi-truststore-file-hostname-verification-policy=WILDCARD
+              "/opt/keycloak/bin/kc.sh" "$@" --spi-truststore-file-file=${TRUSTSTORE_DIR}/keycloak-truststore.jks --spi-truststore-file-password=changeit --spi-truststore-file-hostname-verification-policy=WILDCARD --spi-user-profile-declarative-user-profile-config-file=${USERPROFILE_DIR}/cs-keycloak-user-profile.json
+        - apiVersion: v1
+          data:
+            data:
+              cs-keycloak-user-profile.json: |
+                {
+                  "attributes": [
+                    {
+                      "name": "username",
+                      "displayName": "${variable}{username}",
```

Comment:
Hi @qpdpQ 

I think that we will need to change the displayName to `${username}`.

This is currently displayed from Keycloak UI
<img width="1473" alt="Screenshot 2024-09-23 at 11 59 53 AM" src="https://github.com/user-attachments/assets/74cefbcf-c185-4e28-ae20-c01e3d146c99">